multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -147,12 +147,22 @@ func (c *Cluster) Create(stackBody string) error {
 		return err
 	}
 
+	tags := make([]*cloudformation.Tag, len(c.StackTags))
+	i := 0
+	for k, v := range c.StackTags {
+		key := k
+		value := v
```

Comment:
Why not  `for key, value := range c.StackTags {`?  Or you could just use k and v below.  I'm generally not a fan of introducing variables unless they're necessary.

Similarly I'd prefer to drop `i` and instead write `tags = append(tags, &cloudFormation..Tag{...}`.

Otherwise this looks great to me, thanks :)